playbooks/common/openshift-cluster/upgrades/disable_excluder.yml

```diff
@@ -0,0 +1,14 @@
+---
+- name: Record excluder state and disable
+  hosts: l_oo_all_hosts
+  gather_facts: no
+  tasks:
+  - include_role:
+      name: openshift_excluder
+      tasks_from: status
+      vars:
+        excluder_package_state: latest
+        docker_excluder_package_state: latest
```

Comment:
With the upgrade do we want the latest or ``openshift_version`` latest excluders?